spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -285,6 +285,7 @@
 "LGPL, version 3.0": LGPL-2.1-only
 "LGPL/MIT": LGPL-3.0-only OR MIT
 "LGPLv3 or later": LGPL-3.0-or-later
+"LPGL, see LICENSE file.": LGPL-3.0-only
```

Comment:
I believe we can't do that. The entry directly below maps to 2.1 only, and as we must not make any assumption about the context we cannot map the same thing to different license versions.

Reply:
Yes, I thought about that but for packages with this specific misspelling all come with LGPL-3.0-only